content/engineering/implementing-oidc-sso-having-login-radius-as-identity-provider/index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: "IMPLEMENTING OIDC SSO HAVING LOGIN RADIUS AS IDENTITY PROVIDER"
+date: "2024-05-03"
+#coverImage: "IMPLEMENTING OIDC SSO HAVING LOGIN RADIUS AS IDENTITY PROVIDER.png"
+author: "Sanjay Velu"
+tags: ["SSO", "OIDC", "LoginRadius"]
+description: "In this article, you will implement SSO using OIDC having loginradius as the Identity Provider."
+---
+
+# Introduction
+
+Firstly let us know about what it sso and why one should use it and how we can leverage the login radius as an identity provider to do oidc.
+SSO stands for single sign on . It's an authentication process that allows a user to access multiple applications or systems with one set of login credentials (username and password). Instead of requiring users to log in separately to each application, SSO enables them to log in once and gain access to all the connected systems without needing to re-enter their credentials.
+
+
+## Login Radius SSO
+
+LoginRadius serves as the cornerstone for implementing OpenID Connect (OIDC), primarily functioning as an identity provider. It seamlessly facilitates user authentication, offering a robust login mechanism that ensures secure access to various services. Beyond authentication, LoginRadius plays a pivotal role in safeguarding sensitive data, fortifying it against external threats. Through its comprehensive suite of security measures, LoginRadius ensures that user data remains shielded from unauthorized access, thereby upholding confidentiality and integrity. In essence, LoginRadius not only streamlines the authentication process but also serves as a bulwark, safeguarding invaluable user information from potential breaches.
+
+Once the OIDC application is created, LoginRadius empowers administrators to tailor user claim fields effortlessly. These customizable user claims can be finely tuned through the platform's user-friendly interface. Subsequently, these specified claims are seamlessly integrated into the token, enabling streamlined extraction and utilization within the application ecosystem.
+
+In essence, LoginRadius not only facilitates the setup of OIDC applications but also enhances customization capabilities through its intuitive interface. This ensures efficient management of user claims, ultimately contributing to a more personalized and secure authentication experience.
+
+
+After setting up the oidc app from the login radius dashboard we will use the go -oidc library to further configure our provider and configure the oidc connect.
+
+## Setting up OIDC Application
+
+First go to Admin Dashboard and Navigate to 
```

Comment:
You can add a link of admin console for better context

Reply:
Sure will add it .